docs/faq.md

```diff
@@ -223,6 +448,11 @@ Different airline consolidators handle payments in different ways. In some cases
 
 For Self-Service users, all post-booking services must be handled offline with the consolidator you work with for ticket issuance. In general, these actions can be made while the Passenger Name Record (PNR) is queued for ticketing (before the ticket is issued), though their availability once a ticket has issued depends largely on the consolidator and the clauses of your agreement. 
 
+
+### How can I get a consolidator?
+
+Before requesting a consolidator, please first make sure that you are approved markets for Flight Create Orders. You need it to implement flight booking in Self-Service. Once this is verified, please go to the Support section and get in touch with us using the Contact form.
```

Comment:
Before requesting a consolidator, please first make sure that you are **in one of the** approved markets for Flight Create Orders. You need it to implement flight booking in Self-Service. Once this is verified, please go to the Support section and get in touch with us using the Contact form.